phpBB/adm/style/tooltip.js

```diff
@@ -130,10 +130,17 @@ phpbb.positionTooltip = function ($element) {
 	$element = $element.parent();
 	offset = $element.offset();
 
-	$('#_tooltip_container').css({
-		top: offset.top + 30,
-		left: offset.left - 205
-	});
+	if ($('body').hasClass('rtl')) {
+		$('#_tooltip_container').css({
+			top: offset.top + 30,
+			left: offset.left + 255
+		});
+	} else {
+		$('#_tooltip_container').css({
```

Comment:
is this an actual id? 🤣

Reply:
yes

Reply:
it will help to review you prs if you provide before and after screenshots